test/3.x/app.js

```diff
@@ -134,7 +134,7 @@ app.get('/escape', function(req, res) {
 app.get('/locals', function(req, res) {
   res.locals.person = 'Alan';
   res.render('locals', {
-    layout: false,
+    layout: 'layout_locals',
```

Comment:
Yes, we need to leave existing test. Also, does this mean that `layout_locals` is now a special word and a user cannot use that as the name of their actual layout? If so, this is not desirable.